alma/lib/Factories/ModuleFactory.php

```diff
@@ -55,6 +55,14 @@ public function getModule()
         return \Module::getInstanceByName(ConstantsHelper::ALMA_MODULE_NAME);
```

Comment:
Module name is not a Param - it a getAlmaModule methode and it's an AlmaModuleFactory

Reply:
@Francois-Gomis I'm not sure to understand your request. Can we speak on that in synchroniously